TileSystem/Interfaces/Creation/ICreateTiles.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+using TileSystem.Interfaces.Base;
+
+namespace TileSystem.Interfaces.Creation
+{
+	/// <summary>
+	/// ICreateTiles is used by the level to create tiles
+	/// using a normal creational pattern
+	/// 
+	/// The area, x, y will be where the tile is created
+	/// </summary>
+	public interface ICreateTiles
+	{
+		event EventHandler<TileCreatedArgs> TileCreated;
+		ITile CreateTile(IArea area, int x, int y, string type, string variation, params object[] properties);
```

Comment:
Fixed, now we have a simple interface that is IPosition, which you should implement comparable so we can tell if positionA == positionB